ci/values.yaml

```diff
@@ -1,8 +1,5 @@
 cloud: k3s
 
-image:
```

Comment:
Not sure why we want to test against `latest` but this makes the test non deterministic. I'm removing this so we can use the chart default (latest stable release)